libs/blocks/chart/chart.js

```diff
@@ -579,6 +580,19 @@ export const getOversizedNumberSize = (charLength) => {
   return [fontSize, titleY, subtitleY];
 };
 
+const handleDeferredSections = (section, upNumberStyle) => {
+  const loadLazySection = () => {
+    const upClasses = [];
+    upClasses.push(...[...section.classList].filter((c) => c.endsWith('-up')));
```

Comment:
Why are you pushing into an empty array? Can't you just do
```suggestion
    const upClasses = [...section.classList].filter((c) => c.endsWith('-up'));
```